code/modules/clothing/shoes/colour.dm

```diff
@@ -1,104 +1,113 @@
-/obj/item/clothing/shoes/black
-	name = "black shoes"
-	icon_state = "black"
-	desc = "A pair of black shoes."
+/obj/item/clothing/shoes/color
+	name = "white shoes"
+	icon_state = "white"
+	desc = "A pair of shoes."
 
 	cold_protection = FEET
 	min_cold_protection_temperature = SHOE_MIN_COLD_PROTECTION_TEMPERATURE
 	heat_protection = FEET
 	max_heat_protection_temperature = SHOE_MAX_HEAT_PROTECTION_TEMPERATURE
 
-/obj/item/clothing/shoes/brown
+	var/markings_icon = "shoes_coloring"
+	var/markings_color
+
+/obj/item/clothing/shoes/color/Initialize()
+	. = ..()
+	update_icon()
+
+/obj/item/clothing/shoes/color/on_update_icon()
+	if(markings_color)
+		overlays.Cut()
+		var/mutable_appearance/MA = new()
+		MA.icon = icon
+		MA.icon_state = markings_icon
+		MA.color = markings_color
+		MA.appearance_flags = RESET_COLOR
+		MA.layer = FLOAT_LAYER
+		MA.plane = FLOAT_PLANE
+		overlays += MA
+
+/obj/item/clothing/shoes/color/get_mob_overlay(mob/user_mob, slot)
+	var/image/ret = ..()
+	if(markings_color)
+		var/mutable_appearance/MA = new()
+		MA.icon = ret.icon
+		MA.icon_state = markings_icon
+		MA.color = markings_color
+		MA.appearance_flags = RESET_COLOR
+		MA.layer = FLOAT_LAYER
+		MA.plane = FLOAT_PLANE
+		ret.overlays += MA
```

Comment:
This look rather identical to https://github.com/NebulaSS13/Nebula/pull/430/files#diff-d144d7ca9389f824d315c483c1639f29R19-R28

Reply:
Yeah, sorta thinking of unifying them with some /clothing level handling

Reply:
Moved it to /clothing